typescript/web/src/pages/[workspaceSlug]/datasets/index.tsx

```diff
@@ -58,13 +59,7 @@ const LoadingCard = () => (
       alignItems="center"
       justify="center"
     >
-      <Spinner
-        thickness="4px"
-        speed="0.65s"
-        emptyColor="gray.200"
-        color="#31CECA"
-        size="xl"
-      />
+      <Spinner color="#31CECA" />
```

Comment:
While you're modifying this block, can you make sure that this color cannot come from the theme?

Reply:
Right, I changed it